### PR TITLE
Updates to migrate from travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by KuberDriver**
#### What type of PR is this?
Updates kube-init.sh to enable migration from Travis CI to GitHub Actions.

#### What this PR does / why we need it:
Required for https://github.com/kubernetes-client/python-base/issues/249

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: scripts/kube-init.sh [hunk 1]
```diff
@@ -145,4 +158,4 @@ kubectl get services
 echo "Running tests..."
 set -x -e
 # Yield execution to venv command
-$*
+$*
```
Comment (cclauss) [inline scripts/kube-init.sh:161]: ⛔ means that GitHub is expecting every file to end with one and only one carriage return (\n).

File: scripts/kube-init.sh [hunk 2]
```diff
@@ -32,7 +32,8 @@ setenforce 0
 
 # Mount root to fix dns issues
 # Define $HOME since somehow this is not defined
-HOME=/home/travis
+# Changed from travis to GH Actions agent default user
+HOME=/home/runner
```
Comment (cclauss) [inline scripts/kube-init.sh:36]: Isn't `$HOME` already defined in GHA?  https://docs.github.com/en/actions/learn-github-actions/environment-variables#naming-conventions-for-environment-variables

**Comment by cclauss** [comment]
Probably a good idea to delete `.travis.yml`?

File: scripts/kube-init.sh (hunk 2 above)
Comment (cclauss) [reply scripts/kube-init.sh:36]:
```
- run: echo "$HOME"
/home/runner

```

Comment (KuberDriver) [reply scripts/kube-init.sh]: Yes, thank you! Whilst testing I thought $HOME is required for TOX run but I have tested by removing $HOME and CI runs fine. Updated!

File: scripts/kube-init.sh [hunk 3]
```diff
@@ -99,6 +100,18 @@ sudo minikube start --vm-driver=$MINIKUBE_DRIVER --bootstrapper=kubeadm --logtos
...
+echo "Copy root .minikube to $HOME"
+sudo cp -r /root/.minikube $HOME
+
+echo "Copy root .kube to $HOME"
+sudo cp -r /root/.kube $HOME
+
+sudo chown -R runner:runner $HOME/.kube $HOME/.minikube
+
+# Correct paths to make kubectl accessible without sudo
+sed 's/root/home\/runner/g' $KUBECONFIG > tmp; mv tmp $KUBECONFIG
```
Comment (yliaog) [inline scripts/kube-init.sh:113]: could 'tmp' preexist? what about making a unique temp file name?

File: scripts/kube-init.sh [hunk 4]
```diff
@@ -32,7 +32,8 @@ setenforce 0
 
 # Mount root to fix dns issues
 # Define $HOME since somehow this is not defined
-HOME=/home/travis
+# Changed from travis to GH Actions agent default user
+#HOME=/home/runner
```
Comment (roycaihw) [inline scripts/kube-init.sh:36]: Is this line intentionally commented out?

Comment (cclauss) [reply scripts/kube-init.sh:36]: https://github.com/kubernetes-client/python/pull/1659#pullrequestreview-840025736

File: scripts/kube-init.sh (hunk 3 above)
Comment (KuberDriver) [reply scripts/kube-init.sh:113]: Haven't seen in multiple CI runs and have run successful tests. Details [here](https://github.com/KuberDriver/python/actions)

**Pull request merged**